legacyCore/src/main/res/values-sw300dp/themes.xml

```diff
@@ -0,0 +1,43 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
If you make a common theme with the colors, you can have the parent be that theme so you don't duplicate the colors in here since the screen sizes won't change color

Reply:
yea I just completely forgot about this folder. adjusted and removed the folder.